.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+sudo: false
+node_js:
+  - "stable"
```

Comment:
Could we test some older versions as well? I'm interested in testing 0.12, as we haven't yet updated to a newer Node version at Plaid.